ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -318,6 +318,7 @@ class WebSocketService(
       .mapAsync(1) {
         case msg: TextMessage =>
           msg.toStrict(config.maxDuration).map { m =>
+            logger.info(s"request: ${m.text}")
```

Comment:
I believe the logback config in the SDK is set to info level so this will be shown by default to users. Is that really what you intended?

Reply:
yep, I wanted to capture the queries in case something fails.